### PR TITLE
Feature/Convert to Android Studio project and other miscellaneous updates

### PR DESCRIPTION
**Pull request by bilal-alsharifi**
* Update project to use Android Studio
* Update Gradle and dependencies 
* Update Sdl library to version 4.5
* Ask user explicitly to grant permissions
* Update icon
* Update theme and colors
* Replace images that have Apache license
* Use MBT instead of LBT
* Update package name to com.sdl.mobileweather

### CODE REVIEW

File: MobileWeather/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,37 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.ford.mobileweather"
-    android:versionCode="26"
-    android:versionName="1.7.14	" >
+    package="com.ford.mobileweather">
```
Comment (joeygrover) [inline MobileWeather/app/src/main/AndroidManifest.xml]: I think we should rename this package, and thus all the code packages as well. `com.sdl.mobileweather` should work fine.

File: MobileWeather/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -56,23 +51,37 @@
         </service>
         <service android:name="com.ford.mobileweather.wunderground.WUndergroundService" >
         </service>
-        <service android:name="com.ford.mobileweather.noaa.NOAAService" >
-        </service>
+        <!--<service android:name="com.ford.mobileweather.noaa.NOAAService" >-->
+        <!--</service>-->
```
Comment (joeygrover) [inline MobileWeather/app/src/main/AndroidManifest.xml]: Is there a reason this is commented out?

File: MobileWeather/app/src/main/java/com/ford/mobileweather/activity/MainActivity.java [hunk 3]
```diff
@@ -1,7 +1,6 @@
 package com.ford.mobileweather.activity;
```
Comment (joeygrover) [inline MobileWeather/app/src/main/java/com/ford/mobileweather/activity/MainActivity.java]: The onCreate method should include the call to ` SmartDeviceLinkReceiver.queryForConnectedService(this);`

File: MobileWeather/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (bilal-alsharifi) [reply MobileWeather/app/src/main/AndroidManifest.xml]: Sure, I will update the package name to com.sdl.mobileweather.

File: MobileWeather/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (bilal-alsharifi) [reply MobileWeather/app/src/main/AndroidManifest.xml]: Not really :) I will just remove it completely.

**Pull request merged**